terratag_test.go

```diff
@@ -153,19 +153,28 @@ func testTerraformWithFilter(t *testing.T, version string, filter string) {
 	}
 }
 
-func itShouldGenerateExpectedTerratagFiles(suiteDir string, g *GomegaWithT) {
-	expectedPattern := suiteDir + "/expected/**/*.terratag.tf"
+func itShouldGenerateExpectedTerratagFiles(entryDir string, g *GomegaWithT) {
```

Comment:
There was alot of work towards this function.
When updating a test, I noticed that tests didn't really compare files.
This is now fixed. I had to do some fixes/changes to make all the tests work.